pkgs/by-name/li/libhsts/package.nix

```diff
@@ -9,28 +9,38 @@
   python3,
 }:
 let
-  chromium_version = "90.0.4417.1";
+  chromium_version = "140.0.7324.1";
 
   hsts_list = fetchurl {
     url = "https://raw.github.com/chromium/chromium/${chromium_version}/net/http/transport_security_state_static.json";
-    sha256 = "09f24n30x5dmqk8zk7k2glcilgr27832a3304wj1yp97158sqsfx";
+    hash = "sha256-XV3yZA3Ai4It7S/y4V0h+UtKm8SXm6x1hlITD7jGY9I=";
```

Comment:
in the update.sh file ` -e "0,/sha256/s/sha256 = \".*\"/sha256 = \"$sha256\"/" \` needs to be `hash` and nix-prefetch-url commands needs to be changed as well.

optionally passthru.updateScript can be linked to update.sh